tracer/src/Datadog.Trace/Ci/Test.cs

```diff
@@ -189,6 +189,154 @@ public void SetParameters(TestParameters parameters)
+        }
+
+        if (hostInfo.RuntimeVersion is not null)
+        {
+            SetTag(BenchmarkTestTags.HostRuntimeVersion, hostInfo.RuntimeVersion);
+        }
+
+        if (hostInfo.ChronometerFrequencyHertz is not null)
+        {
+            SetTag(BenchmarkTestTags.HostChronometerFrequencyHertz, hostInfo.ChronometerFrequencyHertz);
+        }
+
+        if (hostInfo.ChronometerResolution is not null)
+        {
+            SetTag(BenchmarkTestTags.HostChronometerResolution, hostInfo.ChronometerResolution);
+        }
+
+        // Job info
+
+        if (jobInfo.Description is not null)
+        {
+            SetTag(BenchmarkTestTags.JobDescription, jobInfo.Description);
+        }
+
+        if (jobInfo.Platform is not null)
+        {
+            SetTag(BenchmarkTestTags.JobPlatform, jobInfo.Platform);
+        }
+
+        if (jobInfo.RuntimeName is not null)
+        {
+            SetTag(BenchmarkTestTags.JobRuntimeName, jobInfo.RuntimeName);
+        }
+
+        if (jobInfo.RuntimeMoniker is not null)
+        {
+            SetTag(BenchmarkTestTags.JobRuntimeMoniker, jobInfo.RuntimeMoniker);
+        }
+    }
+
+    /// <summary>
+    /// Add benchmark data
+    /// </summary>
+    /// <param name="measureType">Measure type</param>
+    /// <param name="info">Measure info</param>
+    /// <param name="statistics">Statistics values</param>
+    public void AddBenchmarkData(BenchmarkMeasureType measureType, string info, BenchmarkDiscreteStats statistics)
+    {
+        var measureTypeAsString = string.Empty;
+        switch (measureType)
+        {
+            case BenchmarkMeasureType.Duration:
+                measureTypeAsString = "duration";
+                break;
+            case BenchmarkMeasureType.RunTime:
+                measureTypeAsString = "run_time";
+                break;
+            case BenchmarkMeasureType.ApplicationLaunch:
+                measureTypeAsString = "application_launch";
+                break;
+            case BenchmarkMeasureType.MeanHeapAllocations:
+                measureTypeAsString = "mean_heap_allocations";
+                break;
+            case BenchmarkMeasureType.TotalHeapAllocations:
+                measureTypeAsString = "total_heap_allocations";
+                break;
+            case BenchmarkMeasureType.GarbageCollectorGen0:
+                measureTypeAsString = "gc_gen0_collections";
+                break;
+            case BenchmarkMeasureType.GarbageCollectorGen1:
+                measureTypeAsString = "gc_gen1_collections";
+                break;
+            case BenchmarkMeasureType.GarbageCollectorGen2:
+                measureTypeAsString = "gc_gen2_collections";
+                break;
+            case BenchmarkMeasureType.MemoryTotalOperations:
+                measureTypeAsString = "memory_total_operations";
+                break;
+            default:
+                return;
+        }
```

Comment:
nit: always switch expressions 🤣 

Alternatively, add `[EnumExtensions]` and call `ToStringFast()` 😉 

```suggestion
        var measureTypeAsString = measureType switch
        {
            BenchmarkMeasureType.Duration => "duration",
            BenchmarkMeasureType.RunTime => "run_time",
            BenchmarkMeasureType.ApplicationLaunch => "application_launch",
            BenchmarkMeasureType.MeanHeapAllocations => "mean_heap_allocations",
            BenchmarkMeasureType.TotalHeapAllocations => "total_heap_allocations",
            BenchmarkMeasureType.GarbageCollectorGen0 => "gc_gen0_collections",
            BenchmarkMeasureType.GarbageCollectorGen1 => "gc_gen1_collections",
            BenchmarkMeasureType.GarbageCollectorGen2 => "gc_gen2_collections",
            BenchmarkMeasureType.MemoryTotalOperations => "memory_total_operations",
            _ => string.Empty,
        }
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3774/commits/83258a6d8396e8ff551e7e941baccff1166d9498